hbase-shell/src/main/ruby/shell/commands/describe_namespace.rb

```diff
@@ -33,14 +33,16 @@ def command(namespace)
 
         formatter.header(['DESCRIPTION'], [64])
         formatter.row([desc], true, [64])
-
-        puts
-        formatter.header(%w[QUOTAS])
         ns = namespace.to_s
-        count = quotas_admin.list_quotas(NAMESPACE => ns) do |_, quota|
-          formatter.row([quota])
+        if admin.exists?(::HBaseQuotasConstants::QUOTA_TABLE_NAME.to_s)
```

Comment:
Mind to provide a UT to address the bug and fix ?  Thanks

Reply:
Thank you @openinx  for looking at this fix.  Added test case.

Reply:
@openinx  I made a mistake in commit message, Should I need to change or it can be taken care during squash?

Reply:
The patch looks good to me now..
> I made a mistake in commit message, Should I need to change or it can be taken care during squash?

You mean which mistake ?   I think I can fix it ( if necessary) during squash..

Reply:
@openinx  This is my commit message "HBASE-23095 fixing ruby checkstyle" but it should be "HBASE-23140 fixing ruby checkstyle".  The mistake in the commit message for JIRA Id.

Reply:
Not a big problem,  Let me handle it. Thanks.